front/src/applications/rollingStockEditor/components/RollingStockEditorForm.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import { LightRollingStock } from 'common/api/osrdEditoastApi';
+import RollingStockCardDetail from 'common/RollingStockSelector/RollingStockCardDetail';
+import { RollingStockInfos } from 'common/RollingStockSelector/RollingStockHelpers';
+import RollingStockParameters from './RollingStockParameters';
+
+export default function RollingStockEditorForm(props: {
+  isEditing: boolean;
+  setIsEditing: React.Dispatch<React.SetStateAction<boolean>>;
+  data: LightRollingStock;
+}) {
+  const { isEditing, setIsEditing, data } = props;
+  const [curvesComfortList, setCurvesComfortList] = useState<string[]>();
```

Comment:
Add a default value to avoid to have to manage the case "undefined"
```suggestion
  const [curvesComfortList, setCurvesComfortList] = useState<string[]>([]);
```

Reply:
done